web/src/app/metalakes/metalake/rightContent/tabsContent/detailsView/DetailsView.js

```diff
@@ -32,14 +32,15 @@ const DetailsView = () => {
       }
     })
   if (paramsSize === 5 && searchParams.get('topic')) {
-    properties.unshift({
-      key: 'replication-factor',
-      value: JSON.stringify(activatedItem?.properties['replication-factor'])?.replace(/^"|"$/g, '')
-    })
-    properties.unshift({
-      key: 'partition-count',
-      value: JSON.stringify(activatedItem?.properties['partition-count'])?.replace(/^"|"$/g, '')
-    })
+    const topicPros = Object.keys(activatedItem?.properties || [])
+      .filter(key => ['partition-count', 'replication-factor'].includes(key))
+      .map(item => {
+        return {
+          key: item,
+          value: JSON.stringify(activatedItem?.properties[item]).replace(/^"|"$/g, '')
+        }
+      })
```

Comment:
I think it can be simplified to the following:
```suggestion
    const topicPros = ['partition-count', 'replication-factor'].map(item => {
      return {
        key: item,
        value: JSON.stringify(activatedItem?.properties[item])?.replace(/^"|"$/g, '')
      }
    })
```